lib/documents/schemas/veterans_support_organisations.json

```diff
@@ -1,4 +1,5 @@
 {
+  "target_stack": "draft",
```

Comment:
This should probably be `live`, unless there are any config changes here that aren't on the [live site](https://www.gov.uk/support-for-veterans) yet.

Reply:
oooh thanks for catching this! i lost it amongst the merge conflicts